src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/AddPaymentAccountsIntegrationTest.java

```diff
@@ -0,0 +1,606 @@
+            e.printStackTrace();
+        }
+        return pbaAddRequest;
+    }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    public void test_add_PaymentAccounts_Failure_Has_Single_Null_PBA_ShouldThrow_400() {
+
+        String request = "{\n"
+                + "    \"paymentAccounts\": [\n"
+                + "        null\n"
+                + "    ]\n"
+                + "}";
+
+        PbaAddRequest pbaAddRequest = convertJsonReqStringToObj(request);
+
+        String userId = createActiveUserAndOrganisation(true);
+
+        Map<String, Object> pbaResponse = professionalReferenceDataClient
+                .addPaymentsAccountsByOrgId(pbaAddRequest, puiFinanceManager, userId);
+
+        ErrorResponse errorResponse = convertJsonToErrorResponseObj(pbaResponse);
+        assertThat(pbaResponse).containsEntry("http_status", "400");
+        assertThat(errorResponse.getErrorDescription()).isEqualTo(ADD_PBA_REQUEST_EMPTY);
+        assertThat(errorResponse.getErrorMessage()).isEqualTo(INVALID_REQUEST.getErrorMessage());
+
+    }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    public void test_add_PaymentAccounts_Failure_has_Empty_Request_ShouldThrow_400() {
+
+        String request = "{\n"
+                + "    \"paymentAccounts\": [\n"
+                + "    ]\n"
+                + "}";
+
+        PbaAddRequest pbaAddRequest = convertJsonReqStringToObj(request);
+
+        String userId = createActiveUserAndOrganisation(true);
+
+        Map<String, Object> pbaResponse = professionalReferenceDataClient
+                .addPaymentsAccountsByOrgId(pbaAddRequest, puiFinanceManager, userId);
+
+        ErrorResponse errorResponse = convertJsonToErrorResponseObj(pbaResponse);
+        assertThat(pbaResponse).containsEntry("http_status", "400");
+        assertThat(errorResponse.getErrorDescription()).isEqualTo(ADD_PBA_REQUEST_EMPTY);
+        assertThat(errorResponse.getErrorMessage()).isEqualTo(INVALID_REQUEST.getErrorMessage());
+
+    }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    public void test_add_PaymentAccounts_Failure_has_Empty_Payload_ShouldThrow_400() {
+
+        String request = "{\n"
+                + "}";
+
+        PbaAddRequest pbaAddRequest = convertJsonReqStringToObj(request);
+
+        String userId = createActiveUserAndOrganisation(true);
+
+        Map<String, Object> pbaResponse = professionalReferenceDataClient
+                .addPaymentsAccountsByOrgId(pbaAddRequest, puiFinanceManager, userId);
+
+        ErrorResponse errorResponse = convertJsonToErrorResponseObj(pbaResponse);
+        assertThat(pbaResponse).containsEntry("http_status", "400");
+        assertThat(errorResponse.getErrorDescription()).isEqualTo(ADD_PBA_REQUEST_EMPTY);
+        assertThat(errorResponse.getErrorMessage()).isEqualTo(INVALID_REQUEST.getErrorMessage());
+
+    }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    public void test_add_PaymentAccounts_Failure_Has_Null_Value_PBA_ShouldThrow_400() {
```

Comment:
This test doesn't look right. The test method name indicates that scenario should throw 400 BAD REQUEST, however, you are asserting on 201 CREATED. I guess this scenario has to throw 400 BAD REQUEST